INSTALL.md

```diff
@@ -19,6 +19,8 @@ is required to compile everything from source.
 - pkg-config
 - Java (only needed for the Java analysis)
 - cmake (only needed for the C/Objective-C analysis)
+- ninja-build (optional, if you wish to use sequential linking when building
+  the C/Objective-C analysis)
```

Comment:
Simpler? Also the tool is called "ninja" not "ninja-build".

```
- the ninja build system (optional, only used when building clang from source)
```

Reply:
Maybe change "cmake" to "CMake" too then?

Reply:
Done!